3_visualize/src/visualize_data.R

```diff
@@ -0,0 +1,59 @@
+# Load Libraries
+library(dplyr)
+library(readr)
+library(stringr)
+
+# Create function to plot data and export as PNG
+
+visualize_data <- function(input_data = '2_process/out/model_summary_results.csv', output_path = '3_visualize/out/figure_1.png') {
+  
+  # Load data
+  model_results <- readr::read_csv(input_data)
+  
+  # create .png file, though this is probably redundant and will be an argument
+  # png_path <- '3_visualize/out/figure_1.png'
+  png(file = output_path, width = 8, height = 10, res = 200, units = 'in')
+  par(omi = c(0,0,0.05,0.05), mai = c(1,1,0,0), las = 1, mgp = c(2,.5,0), cex = 1.5)
+  
+  # Plot data, though...what is NA?
+  plot(NA, NA, xlim = c(2, 1000), ylim = c(4.7, 0.75),
+       ylab = "Test RMSE (°C)", xlab = "Training temperature profiles (#)", log = 'x', axes = FALSE)
+  
+  n_profs <- c(2, 10, 50, 100, 500, 980)
+  
+  axis(1, at = c(-100, n_profs, 1e10), labels = c("", n_profs, ""), tck = -0.01)
+  axis(2, at = seq(0,10), las = 1, tck = -0.01)
+  
+  # slight horizontal offsets so the markers don't overlap:
+  offsets <- data.frame(pgdl = c(0.15, 0.5, 3, 7, 20, 30)) %>%
+    mutate(dl = -pgdl, pb = 0, n_prof = n_profs)
+  
+  
+  for (mod in c('pb','dl','pgdl')){
+    mod_data <- filter(eval_data, model_type == mod)
```

Comment:
I think `eval_data` isn't read in anywhere yet. Should it be replaced with `model_results` instead?